app/controllers/concerns/controller_helpers.rb

```diff
@@ -104,9 +104,9 @@ def return_to_if_present
     end
   end
 
-  def translation(key, scope: nil)
+  def translation(key, scope: nil, **kwargs)
```

Comment:
Can you still pass in the `scope` keyword argument here?

Reply:
yup — any other kwargs are captured by `**kwargs`